### PR TITLE
EMQX Management README

### PR DESCRIPTION
**Pull request by sstrigler**
Fixes [EMQX-9253](https://emqx.atlassian.net/browse/EMQX-9253)

Make sure to target release-50 branch if this PR is intended to fix the issues for the release candidate.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [-] Added tests for the changes
- [-] Changed lines covered in coverage report
- [-] Change log has been added to `changes/{ce,ee}/(feat|perf|fix)-<PR-id>.en.md` files
- [x] For internal contributor: there is a jira ticket to track this change
- [-] If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up
- [-] Schema changes are backward compatible

### CODE REVIEW

File: apps/emqx_management/README.md [hunk 1]
```diff
@@ -1,12 +1,44 @@
...
+* alarms,
+* API Keys,
+* banned clients, users or hosts,
+* connected clients including their topic subscriptions,
+* cluster configurations,
+* configuration of MQTT listeners,
+* node configuration,
+* custom plugins,
+* fixed subscriptions,
+* and topics.
```
Comment (zmstone) [inline apps/emqx_management/README.md:24]: Not sure what's `and topics`.

```suggestion
* Alarms
* API Keys
* Banned clients, users or hosts
* Clients (and sessions) including their topic subscriptions
* Configurations
* Manage plugins
* Fixed subscriptions
```

File: apps/emqx_management/README.md [hunk 2]
```diff
@@ -1,12 +1,44 @@
...
+
+Moreover it lets you
```
Comment (zmstone) [inline apps/emqx_management/README.md:26]: wonder what makes an item listed above or below .

File: apps/emqx_management/README.md [hunk 3]
```diff
@@ -1,12 +1,44 @@
...
+
+* modify hot and non-hot updatable configuration values,
+* publish messages, as well as bulk messages,
+* create trace files,
+* and last but not least monitor system status.
+
+## Implementation Notes
+
+API endpoints are implemented using the `minirest` framework in combination with
+`hoconsc` and `emqx_dashboard_swagger`.
```
Comment (zmstone) [inline apps/emqx_management/README.md:36]:
```suggestion
API endpoints are implemented using the `minirest` framework in combination with
HOCON schema and OpenAPI 3.0 specifications.
```

File: apps/emqx_management/README.md (hunk 2 above)
Comment (sstrigler) [reply apps/emqx_management/README.md:26]: You don't manage "metrics" for instance. And I thought it just makes it easier to read when you mix things up a bit.

File: apps/emqx_management/README.md (hunk 1 above)
Comment (sstrigler) [reply apps/emqx_management/README.md:24]: That you can manage `topics`.

File: apps/emqx_management/README.md [hunk 4]
```diff
@@ -1,12 +1,41 @@
...
-default application see:
-header:
-authorization: Basic YWRtaW46cHVibGlj
+* Alarms
+* API Keys
+* Banned clients, users or hosts
+* Clients (and sessions) including their topic subscriptions
+* Configurations
+* Manage plugins
+* Fixed subscriptions
```
Comment (sstrigler) [inline apps/emqx_management/README.md:21]:
```suggestion
* Fixed subscriptions
* Topics
```

**Review by zmstone — APPROVED** [review]

**Pull request merged**